app/views/timeline_entries/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name} timeline entries",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Add timeline entry"
+    },
+  ]
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, formatted_title(@coronavirus_page) %>
+<% content_for :context, "Add timeline entry" %>
+<% if @timeline_entry.errors.any? %>
+  <%= render "shared/sub_sections/form_errors", resource: @timeline_entry %>
+<% end %>
+
+<div class="covid-edit-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @timeline_entry,
+                  url: coronavirus_page_timeline_entries_path,
+                  local: true do %>
```

Comment:
I did some reading about this, and wow what a super annoying quirk! The good news is that Rails recognised this is weird too and in [Rails 6.1 this is going to default to be a local form](https://github.com/rails/rails/pull/40946) and you have to specify remote to change that.

Rather than specifying `local` here on this form I'd suggest we change config/application.rb to have `config.action_view.form_with_generates_remote_forms = false` so that this configuration is applied globally rather than needed on individual forms. We can leave a note with that to say we can remove that line once we've adopted Rails 6.1 config.

Reply:
This is much nicer! I'll add a comment to config explaining why it's there.